README.md

```diff
@@ -1,2 +1,15 @@
 # blog
 Blog posts from PH for PH and the rest of the world :)
+
+Formating of blog posts:
+
+When you first make a blog post make sure to have a JSON header stating your name, title of your blog post, date, tags, and publication. 
+The JSON should also be contained in brackets. An example of this is down below. 
+
+{ "author": "Riyan Imam", "title": "HackATL: Busy Making Businesses!", "date": "2016-11-08T23:35:24Z", "tags": ["all"], "published": true
+
+}
+
+After you finished setting up your JSON header make sure to have three dashes and write your content of your blog post in Markdown. At the end of your blog post you should a have a dash and your full name written. An example of this is down below. 
+
+-Abele Amsalu
```

Comment:
You don't really need that. It's just a think Titan does that I'll start regulating because I think it looks bad